packages/asgardex-bitcoin/src/client.ts

```diff
@@ -299,7 +324,7 @@ class Client implements BitcoinClient, AsgardexClient {
 
   async transfer({ asset, amount, recipient, memo, feeRate }: TxParams): Promise<string> {
```

Comment:
[suggestion] Make `asset?: Asset` optional in `TxParams`. Because we won't send different assets in some cases, currently just for BNC or ETH, but not for BTC. 

With this change, we will have
```
async transfer({ asset = AssetBTC, amount, recipient, memo, feeRate }: TxParams): Promise<string>
```
which will simplified things by calling `transfer` function...